### PR TITLE
Performance: Use `webm`

### LINKED ISSUE #1466: Website performance: GIFs/mp4s should be served as MPEG4/WebM
Use video formats for animated content. GIFs should be served as MPEG4/WebM

[Affected pages](https://docs.google.com/spreadsheets/d/1vKryTY7tbtJsCBSosCiPnbLY2WHBbvC3gvpYi_LLpLA/edit#gid=0)

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
> fixes #1466
---

[**` Live Preview `**](https://deploy-preview-2921--gitpod-kumquat.netlify.app/)

_TL;DR;_

- This PR updates all (`*.gif` & `*.mp4`) assets to `*.webm`.
- This PR Deletes all outdated/ unused and other `.gif`/ `.mp4` files
- This PR introduces `controls` over the video to play, pause, move to seek. So that it will be easy for the user to move ahead, and play/ pause to any extent. Because we have a lot of longer videos (>10s) 

### Benefits

- Faster Loading Time (a lot of internal benefits on SEO & Performance)
- Same Video Quality with reduced asset size.
- Video Controls to pause/play/forward the video (more accessible & usable for videos)

---

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2921"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2921"></a>

### CODE REVIEW

**Review by Cahllagerfeld — COMMENTED** [review]
grafik
On a couple of pages in the editors, a clip is missing a source, e.g Pycharm

**Comment by Siddhant-K-code** [comment]
Marked it `Urgent` as it affects a lot of files & will leads to merge conflicts (_if delayed_)

File: gitpod/docs/integrations/jetbrains-gateway.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Since [JetBrains Gateway](https://www.jetbrains.com/remote-development/gateway/)
 To start a workspace using JetBrains from your browser, you need to first set your preferred JetBrains IDE as your user preference.
 
 <figure>
-<video playsinline autoplay no-controls loop muted class="shadow-medium w-full rounded-xl max-w-3xl mt-x-small" alt="Updating Gitpod IDE preferences to {title}" src="/images/editors/select-jetbrains-ide.mp4"></video>
+<video controls playsinline autoplay no-controls loop muted class="shadow-medium w-full rounded-xl max-w-3xl mt-x-small" alt="Updating Gitpod IDE preferences to {title}" src="/images/editors/select-jetbrains-ide.webm" type="video/webm"></video>
```
Comment (Cahllagerfeld) [inline gitpod/docs/integrations/jetbrains-gateway.md:52]: we should remove `no-controls` if we want to have them

**Review by Cahllagerfeld — APPROVED** [review]
Other than that you can merge if you tested everything 👍

**Comment by chrifro** [comment]
Nice ✨ Do you know how much this will impact the performance? 

>Video Controls to pause/play/forward the video (more accessible & usable for videos)

I couldn't see any differences in the preview. What am I missing?

**Comment by Siddhant-K-code** [comment]
> Do you know how much this will impact the performance?

Not sure about the exact numbers, but the key reason is [this comparison blog](https://techstacker.com/why-webm-is-superior-to-gif-video-comparison/FR6xLr2zHn9uSTTsH/#:~:text=Facts%20about%20the%20WebM%20format) & [this doc. by Google Devs](https://web.dev/efficient-animated-content/#why-you-should-replace-animated-gifs-with-video)

> I couldn't see any differences in the preview. What am I missing?

When you hover over Video it will show you something like this:

image

File: gitpod/docs/integrations/jetbrains-gateway.md (hunk 1 above)
Comment (Siddhant-K-code) [reply gitpod/docs/integrations/jetbrains-gateway.md]: Resolved in [`5534362`](https://github.com/gitpod-io/website/pull/2921/commits/5534362697a89112a10203f733ff064b7f5bcbbd)

**Comment by filiptronicek** [comment]
Just noticed a lot of the gifs are actually larger as `webm`s 👀 @Siddhant-K-code 
image
image

**Comment by Siddhant-K-code** [comment]
> Just noticed a lot of the gifs are actually larger as `webm`s 👀 @Siddhant-K-code

I am using `ffmpeg` to convert `.gif` file to a `.webm` file with a following command:


```sh
ffmpeg -i input.gif -c vp9 -b:v 0 -crf 10 output.webm
```

I am using `crf` to specify the quality, which ranges between 0-63, lower means better quality. I tried to increase it but suffers the pixel quality.

**Comment by Siddhant-K-code** [comment]
For Now, I am merging this PR as looks good. We can re-iterate in follow-up PRs if there will be any changes
